docs/frequently-asked-questions.rst

```diff
@@ -66,13 +66,6 @@ arguments for you.
 See `ping.sol <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/45_ping.sol>`_ and
 `pong.sol <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/45_pong.sol>`_.
 
-What happens if you send ether along with a function call to a contract?
-========================================================================
-
-It gets added to the total balance of the contract, just like when you send ether when creating a contract.
```

Comment:
I'd add this to https://solidity.readthedocs.io/en/v0.5.2/structure-of-a-contract.html#functions. Or is this explained somewhere else?

Reply:
This is the current functions sections:
> Function Calls can happen internally or externally and have different levels of visibility towards other contracts. Functions accept parameters and return variables to pass parameters and values between them.

I'd say it would be reasonable to include something along the lines "Functions can also have special attributes (modifiers?), like `payable` <link here>"

Reply:
@axic In the section below this paragraph is a section about function modifiers. Thing is, in your suggestion you mention linking to a list of function modifiers, and I agree, there isn't really a list (in one place) of what's available. So I think creating one is a good idea, but perhaps out of scope of this PR?

Reply:
@ChrisChinchilla That sounds like a good idea. Let's create an issue for that and merge this one :)